pinot-controller/src/main/resources/app/components/Table.tsx

```diff
@@ -381,6 +381,11 @@ export default function CustomizedTables({
 
   const makeCell = (cellData, rowIndex) => {
     if (Object.prototype.toString.call(cellData) === '[object Object]') {
+      // render custom table cell
```

Comment:
great addition. this has been getting quite complex supporting every table in pinot